SPECS/llvm/llvm.spec

```diff
@@ -96,9 +102,14 @@ ninja check-all
 %files devel
 %{_libdir}/*.a
 %{_libdir}/cmake/*
+%{_libdir}/LLVMgold.so
+%{_libdir}/bfd-plugins/LLVMgold.so
 %{_includedir}/*
 
 %changelog
+* Fri Apr 5 2024 Mykhailo Bykhovtsev <mbykhovtsev@microsoft.com> - 18.1.2-2
+- Added LLVMgold.so files to the main and devel packages
```

Comment:
Why to both of them? Also, where did they come from?

Reply:
I am not sure, they are provided in both packages in Fedora

Reply:
I don't think that's correct. Fedora 40 seems to have a completely different way of splitting the files across subpackages. The `LLVMgold.so` files seem to be only present in their `llvm-libs` package. With the way we've got these files separate, I'd remove the ones in `llvm-devel`, since that package already pulls in `llvm`.

Reply:
Removed